httpie/cli/requestitems.py

```diff
@@ -107,6 +111,10 @@ def process_query_param_arg(arg: KeyValueArg) -> str:
     return arg.value
 
 
+def process_embed_query_param_arg(arg: KeyValueArg) -> str:
+    return load_text_file(arg).rstrip('\n')
```

Comment:
> But what if I want to send some trailing line breaks (perhaps that’s the very reason I use a file instead of ==)? thinking

So we should only strip the last one (if there is), right?

Reply:
I don't think we should strip. Just having to explain/test the logic makes it rather complex.

Reply:
Or is it sort of standard behavior?

Reply:
> I don't think we should strip. Just having to explain/test the logic makes it rather complex.

I agree that it is unusual (especially compared to other `@` operators), but newline is not generally what they meant. And if they don't know how to write into a file in any other way (e.g `echo 'blabla' > file`), then they wouldn't be able to use this feature.

Reply:
Right, this will become even more pronounced with headers from file, where new line are not allowed.